aten/src/ATen/ConjugateFallback.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <torch/library.h>
+
+namespace at {
+
+    // This fallback effectively takes all tensors in the stack
+    // with their conjugate bit set, and runs conjugation on them
+    void conjugateFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack);
```

Comment:
add a `CAFFE2_API` to the beginning of this function, that will fix your linker error. (We hide symbols by default to make our behavior like Windows, and that means you need to explicitly annotate functions which will be accessible from other files).

However, even better would be to directly register the fallback in `ConjugateFallback.cpp` using a `TORCH_LIBRARY_IMPL` block, rather than in the test; after all, you want the registration to be available for all code!